working/augmentations/feature-specification.md

```diff
@@ -616,6 +604,20 @@ augment enum E { v5 }
 
 *The resulting enum has values `v1`, `v2`, `v3`, `v4`, and `v5`, in that order.*
 
+*A consequence of application order is that an augmentation in one part file can
+augment an introductory declaration in a sibling part file. This is intuitively
+confusing. The augmentation is referring to a declaration that can't be found
```

Comment:
> This is intuitively confusing

No, it's not!
(Is so! Is not!)

That depends very much on the intuition you depend on for understanding how this works.

If we don't tell people that the "below" order matters, they shouldn't _start_ with that intuition. If they know that the "After" order is important, and start thinking of part directives as defining an order, that's the order they'll use to think about it. Then it's not confusing that a *later* sibling can augment  an earlier declaration.

Maybe say "Can be confusing". Definitely drop "intuitively". No two persons can agree on what is "intuitive".